### PR TITLE
prepare 804 release

### PR DESCRIPTION
**Pull request by jospoortvliet**
@cmonteroluque I had to turn the long doc from carlos into a short changelog. Would be nice if next time the most important items are on top with a human description or something... Or, I need more time ;-)

@karlitschek can you check, I haven't done all the changes to the website for a release yet, not sure what else needs to be done, if anything...

### CODE REVIEW

**Comment by jospoortvliet** [comment]
ok, looking at earlier commits, this seems good enough. Going to push to live and to admin.

**Comment by carlaschroder** [comment]
NOOO WAIT

**Comment by carlaschroder** [comment]
Made you look :)

File: page-changelog.php [hunk 1]
```diff
@@ -1,7 +1,64 @@
 <div class="page-header">
 	<h1>ownCloud Server Changelog</h1>
 </div>
+<h3>Version 8.0.4 <small>June 5th 2015</small></h3>
```
Comment (RealRancor) [inline page-changelog.php]: Really released on June 5th? :grinning:

**Comment by jospoortvliet** [comment]
@RealRancor fixed with #573

**Pull request merged**